test/test_cohort.py

```diff
@@ -163,126 +222,124 @@ async def setUp(self):
 
         self.sA = await self.samplel.upsert_sample(get_sample_model('A'))
         self.sB = await self.samplel.upsert_sample(get_sample_model('B'))
-        self.sC = await self.samplel.upsert_sample(get_sample_model('C', 'saliva', 'exome', 'long-read', 'ONT'))
+        self.sC = await self.samplel.upsert_sample(
+            get_sample_model('C', 'saliva', 'exome', 'long-read', 'ONT')
+        )
 
         self.sgA = sequencing_group_id_format(self.sA.sequencing_groups[0].id)
+        self.sgA_raw = self.sA.sequencing_groups[0].id
         self.sgB = sequencing_group_id_format(self.sB.sequencing_groups[0].id)
+        self.sgB_raw = self.sB.sequencing_groups[0].id
         self.sgC = sequencing_group_id_format(self.sC.sequencing_groups[0].id)
+        self.sgC_raw = self.sC.sequencing_groups[0].id
 
     @run_as_sync
     async def test_create_cohort_by_sgs(self):
         """Create cohort by selecting sequencing groups"""
         result = await self.cohortl.create_cohort_from_criteria(
             project_to_write=self.project_id,
-            author='bob@example.org',
             description='Cohort with 1 SG',
             cohort_name='SG cohort 1',
             dry_run=False,
             cohort_criteria=CohortCriteria(
```

Comment:
Why not use the construct the CohortCriteriaInternal here directly, then you don't need the extra map? (You store the raw sgV_raw value already

Reply:
Is there a point of view that doing this exercises the CohortCriteria <-> CohortCriteriaInternal machinery btw? Or is that sufficiently exercised elsewhere… either way, only one test case would need to keep the current circumlocutions…

Reply:
For sure, and yeah I’d leave it to one, or move it to its own test so it’s easier to debug

Reply:
Thanks for the suggestion. Done, with a separate test for the conversions.